packages/generator-langium/templates/packages/language/test/linking.test.ts

```diff
@@ -2,8 +2,7 @@ import { afterEach, beforeAll, describe, expect, test } from "vitest";
 import { EmptyFileSystem, type LangiumDocument } from "langium";
 import { expandToString as s } from "langium/generate";
 import { clearDocuments, parseHelper } from "langium/test";
-import { create<%= LanguageName %>Services } from "../../src/language/<%= language-id %>-module.js";
-import { Model, isModel } from "../../src/language/generated/ast.js";
+import { create<%= LanguageName %>Services, isModel, Model } from "<%= language-id %>-language";
```

Comment:
It hasn't been changed yet:

```suggestion
import type { Model } from "<%= language-id %>-language";
import { create<%= LanguageName %>Services, isModel } from "<%= language-id %>-language";
```

Reply:
Overlooked that one, too.

Reply:
Done ⬇️